TestProject/ClientApp/src/components/CommentInput.js

```diff
@@ -0,0 +1,44 @@
+﻿import React, { useState } from 'react';
+import { NotificationContainer, NotificationManager } from 'react-notifications';
+import 'react-notifications/lib/notifications.css';
+
+function CommentInput({ postId, updatePost }) {
+    const [comment, setComment] = useState("");
+
+    function handleCommentChange(value){
+        setComment(value);
+    }
+
+    function handleSubmit() {
+        fetch("/comments/create", {
+            method: 'post',
+            headers: {
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({text: comment, postId})
+        }).then((response) => {
+            if (response.ok) {
+                updatePost();
+                setComment("");
+            }
+            else {
+                NotificationManager.error("Couldn't create comment");
+            }
+        }).catch(() => {
+            NotificationManager.error("Couldn't create comment");
+        });
```

Comment:
You use this construction everywhere in the application, consider extracting it into some kind of service.